themes/base/accordion.css

```diff
@@ -21,3 +21,14 @@
 	border-top: 0;
 	overflow: auto;
 }
+
+/* RTL support */
+.ui-accordion-rtl {
+	direction: rtl;
```

Comment:
That is correct for the current case. But it would be required if there is a parent widget with `dir ='rtl'` & its child widget exists somewhere in the DOM with no `dir `attribute (e.g. autocomplete & menu).
I know it doesn't make any sense her, but I added it in case we need it at some point & for the mirroring support to be consistent across all the widgets.